bin/commands/init/security/index.ts

```diff
@@ -0,0 +1,173 @@
+  const securityUsersZowe=zoweConfig.zowe.setup.security?.users?.zowe || std.getenv('ZWE_PRIVATE_DEFAULT_ZOWE_USER');
+  const securityUsersZis=zoweConfig.zowe.setup.security?.users?.zis || std.getenv('ZWE_PRIVATE_DEFAULT_ZIS_USER');
+  const securityStcsZowe=zoweConfig.zowe.setup.security?.stcs?.zowe || std.getenv('ZWE_PRIVATE_DEFAULT_ZOWE_STC');
+  const securityStcsZis=zoweConfig.zowe.setup.security?.stcs?.zis || std.getenv('ZWE_PRIVATE_DEFAULT_ZIS_STC');
+  const securityStcsAux=zoweConfig.zowe.setup.security?.stcs?.aux || std.getenv('ZWE_PRIVATE_DEFAULT_AUX_STC');
+
+
+  // prepare ZWESECUR JCL
+  common.printMessage(`Modify ZWESECUR`);
+  const spaceReplacer = new RegExp('\ ', 'g');
+  let tmpfile=fs.createTmpFile(`zwe ${std.getenv('ZWE_CLI_COMMANDS_LIST')}`.replace(spaceReplacer, '-'));
+  const tmpdsm=zosdataset.createDatasetTmpMember(jcllib, `ZW${strftime("%H%M")}`);
+  common.printDebug(`- Copy ${prefix}.${std.getenv('ZWE_PRIVATE_DS_SZWESAMP')}(ZWESECUR) to ${tmpfile}`);
+  // cat "//'IBMUSER.ZWEV2.SZWESAMP(ZWESECUR)'" | sed "s/^\\/\\/ \\+SET \\+PRODUCT=.*\\$/\\/\\         SET  PRODUCT=ACF2         * RACF, ACF2, or TSS/"
+  const zwesecurResult = shell.execOutSync('sh', '-c', ` cat "//'${prefix}.${std.getenv('ZWE_PRIVATE_DS_SZWESAMP')}(ZWESECUR)'"`);
+  if (zwesecurResult.rc == 0) {
+    common.printDebug(`  * Succeeded`);
+    common.printTrace(`  * Exit code: ${zwesecurResult.rc}`);
+    common.printTrace(`  * Output:`);
+    if (zwesecurResult.out) {
+      common.printTrace(stringlib.paddingLeft(zwesecurResult.out, "    "));
+    }
+
+
+//this doesnt appear to have worked.
+    
+    xplatform.storeFileUTF8(tmpfile, xplatform.AUTO_DETECT,
+                          zwesecurResult.out
+                          .replace(/\n\/\/\s+SET\s+PRODUCT=.*\n/, `\n//         SET  PRODUCT=${securityProduct}\n`)
+                          .replace(/\n\/\/\s+SET\s+ADMINGRP=.*\n/, `\n//         SET  ADMINGRP=${securityGroupsAdmin}\n`)
+                          .replace(/\n\/\/\s+SET\s+STCGRP=.*\n/, `\n//         SET  STCGRP=${securityGroupsStc}\n`)
+                          .replace(/\n\/\/\s+SET\s+ZOWEUSER=.*\n/, `\n//         SET  ZOWEUSER=${securityUsersZowe}\n`)
+                          .replace(/\n\/\/\s+SET\s+ZISUSER=.*\n/, `\n//         SET  ZISUSER=${securityUsersZis}\n`)
+                          .replace(/\n\/\/\s+SET\s+ZOWESTC=.*\n/, `\n//         SET  ZOWESTC=${securityStcsZowe}\n`)
+                          .replace(/\n\/\/\s+SET\s+ZISSTC=.*\n/, `\n//         SET  ZISSTC=${securityStcsZis}\n`)
+                          .replace(/\n\/\/\s+SET\s+AUXSTC=.*\n/, `\n//         SET  AUXSTC=${securityStcsAux}\n`)
+                          .replace(/\n\/\/\s+SET\s+HLQ=.*\n/, `\n//         SET  HLQ=${prefix}\n`)
+                          .replace(/\n\/\/\s+SET\s+SYSPROG=.*\n/, `\n//         SET  SYSPROG=${securityGroupsSysprog}\n`));
```

Comment:
Seems to work fine on our system.

I was able to broke this only by manual editing of source `SAMP(ZWESECUR)`, for example by adding label to `SET` statement: `//PRODUCT    SET  PRODUCT=RACF` (but this is too obvious, that it will not work).